core/src/test/scala/org/apache/spark/scheduler/DAGSchedulerSuite.scala

```diff
@@ -540,6 +540,42 @@ class DAGSchedulerSuite extends SparkFunSuite with LocalSparkContext with TimeLi
     assert(mapStatus2(2).location.host === "hostB")
   }
 
+  test("[SPARK-32003] All shuffle files for executor should be cleaned up on fetch failure") {
```

Comment:
Thank you for adding this UT. BTW, since the recommendation from https://spark.apache.org/contributing.html is `SPARK-32003: `, it would be great if this follows the guideline.

Reply:
No problem. I'll change to use "SPARK-32003: ...". There were several different styles in the file; I didn't realize there was an official/preferred style. The first couple of tests in the file had the style I used.